RecoHGCal/TICL/plugins/PatternRecognitionbyCLUE3D.h

```diff
@@ -94,13 +103,24 @@ namespace ticl {
     void dumpTiles(const TILES&) const;
 
     std::vector<ClustersOnLayer> clusters_;
+    std::vector<float> layersPosZ_;
 
     edm::ESGetToken<CaloGeometry, CaloGeometryRecord> caloGeomToken_;
     const double criticalDensity_;
+    const double criticalSelfDensity_;
     const int densitySiblingLayers_;
     const double densityEtaPhiDistanceSqr_;
-    const double densityOnSameLayer_;
+    const double densityXYDistanceSqr_;
+    const double kernelDensityFactor_;
+    const bool densityOnSameLayer_;
+    const bool nearestHigherOnSameLayer_;
+    const bool useAbsoluteProjectiveScale_;
+    const bool useClusterDimensionXY_;
+    const bool rescaleDensityByZ_;
+    ;
```

Comment:
What is this `;` for?

Reply:
A left-over after editing, I guess.

Reply:
Would you mind to clean it up? So we don't need to remember to make it happens in (near) future. I can also sign if you have a plan to clean it with following PR, if any, in a short time.